ui/src/timeMachine/utils/queryBuilder.ts

```diff
@@ -95,35 +94,18 @@ export function buildQuery(builderConfig: BuilderConfig): string {
   const {functions} = builderConfig
 
   let query: string
-  const helper = isFlagEnabled('queryBuilderGrouping')
-    ? buildQueryHelperButWithGrouping
-    : buildQueryHelper
-
   if (functions.length) {
-    query = functions.map(f => helper(builderConfig, f)).join('\n\n')
+    query = functions
+      .map(f => buildQueryFromConfig(builderConfig, f))
+      .join('\n\n')
   } else {
-    query = helper(builderConfig, null)
+    query = buildQueryFromConfig(builderConfig, null)
   }
 
   return query
 }
 
-function buildQueryHelper(
-  builderConfig: BuilderConfig,
-  fn?: BuilderConfig['functions'][0]
-): string {
-  const [bucket] = builderConfig.buckets
-  const tagFilterCall = formatTagFilterCall(builderConfig.tags)
-  const {aggregateWindow} = builderConfig
-  const fnCall = fn ? formatFunctionCall(fn, aggregateWindow.period) : ''
-
-  const query = `from(bucket: "${bucket}")
-  |> range(start: ${OPTION_NAME}.${TIME_RANGE_START}, stop: ${OPTION_NAME}.${TIME_RANGE_STOP})${tagFilterCall}${fnCall}`
-
-  return query
-}
-
-function buildQueryHelperButWithGrouping(
```

Comment:
this diff is wonky. i renamed `buildQueryHelperButWithGrouping` -> `buildQueryFromConfig` and removed `buildQueryHelper` which I thought could have been named a little clearly (hence `buildQueryFromConfig`)